src/main/java/com/ericsson/ei/frontend/EIRequestsController.java

```diff
@@ -93,7 +93,7 @@ public void setUseSecureHttp(boolean useSecureHttp) {
      *
      */
     @CrossOrigin
-    @RequestMapping(value = "/auth/*", method = RequestMethod.GET)
+    @RequestMapping(value = {"/auth/login", "/auth/logout"}, method = RequestMethod.GET)
     public ResponseEntity<String> getAuthRequests(Model model, HttpServletRequest request) {
         String eiBackendAddressSuffix = request.getServletPath();
```

Comment:
It would be good to log the Login/Logout request, so we can debug the traffic. Maybe not the content of the request(with user/pass), but that a login request has been made from EI Fronted UI towards and also for the response.

Reply:
For login/logout request we have next log (same as for other requests):

LOG.info("Got HTTP Request with method GET.\nUrlSuffix: " + eiBackendAddressSuffix +  "\nForwarding Request to EI Backend with url: " + newRequestUrl);

Is it enough?